ui/app/AppLayouts/Onboarding2/KeycardCreateProfileFlow.qml

```diff
@@ -18,7 +20,7 @@ SQUtils.QObject {
     required property int addKeyPairState
     required property int keycardPinInfoPageDelay
 
-    required property var getSeedWords
+    required property var generateMnemonic
```

Comment:
We don't need to pass the function, we could just generate it outside, and pass a string here

Reply:
But this component calls `generateMnemonic` in specific moment between screens:
https://github.com/status-im/status-desktop/blob/5068f85bc17f7cbf073a20206886c25ca3567e69/ui/app/AppLayouts/Onboarding2/KeycardCreateProfileFlow.qml#L128-L133

I guess that's why it's a function rather than passing the value right away.

Anyway, I'd prefer not to refactor the logic at the same time. The intention of this PR is to change the mnemonic format and accordingly rename the method.